cmd/gaia/cli_test/cli_test.go

```diff
@@ -90,19 +91,19 @@ func TestGaiaCLIDeclareCandidacy(t *testing.T) {
 
 	// TODO timeout issues if not connected to the internet
```

Comment:
Why - what requires internet? All connections should be local, presumably?

Reply:
This is a bug presumably in Tendermint/CLI testing which should be addressed - separate from this PR though